docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -339,49 +301,50 @@ func main() {
+Imagine for the purpose of this tutorial that the `fake` motor we are adding controls a conveyor belt in front of your mock arm on an assembly line.
 
-### What is a part?
+### What is a sub-part?
 
-A _{{< glossary_tooltip term_id="robot" text="robot" >}}_ in Viam consists of one or more computers combined into one logical unit.
-The bounds of a robot are usually pretty clear, but can be subjective.
-However, it's possible with Viam to create a robot that is made up of multiple computers.
-Each of these computer-controlled units is referred to as a _{{< glossary_tooltip term_id="part" text="part" >}}_.
-Most simple robots will have only one part, but they can have as many parts as needed.
+Usually, when building a {{< glossary_tooltip term_id="robot" text="robot" >}}, you pick out a [single-board computer](/components/board/) like the [Jetson Nano](/components/board/jetson/) or [Raspberry Pi](/components/board/pi/).
+You follow the instructions in the **Setup** tab to install `viam-server` on your [board](/components/board/), and you start operating your robot with that computer, adding the [components](/components/) and [services](/services/) you want to use to that `viam-server` instance.
 
-Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
-You can access any sub-part either directly, or through any part above it in the tree.
-Each part runs a single `viam-server` instance.
+By utilizing {{< glossary_tooltip term_id="part" text="parts" >}}, you can expand upon this, chaining multiple computers together to build your robot with Viam:
 
-## Configure a sub-part in the Viam app
+- Each individual computer-controlled unit of a robot is called a “{{< glossary_tooltip term_id="part" text="part" >}}” in Viam.
+- Typically, simple robots have just one part, but you can have as many parts as your project requires.
+- Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
+- You can access any sub-part either directly, or through any part above it in the tree.
+- Each part runs a single `viam-server` instance.
 
-You will be creating a mock sub-part to control a motor.
-This could be anything, but imagine for the purpose of this tutorial that this motor controls a conveyor belt in front of your mock arm on an assembly line.
+### Add a new sub-part in the Viam app
 
 On your robot's page on the Viam app, click on the dropdown next to the main part, name your part and click **Add new**.
 
 ![Screenshot of the Viam app with a dropdown below the main part. 'SubPart' is written in the textbox.](/tutorials/build-a-mock-robot/part-menu.png)
 
-Navigate to your new sub-part's **Config** tab and create a new motor using the **fake** model.
+Navigate to your new sub-part's **Config** tab and create a new motor using the `fake` model.
 Name it `"motor2"`.
```

Comment:
```suggestion
Navigate to your new sub-part's **Config** tab and create a new motor:

Click **Create component** in the lower-left corner of the page.
Select type `motor` and model `fake`.
Enter `motor2` as the name and click **Create**.
```
[opt] Or some version of this... We spell it out for users in all other places, so I think we should here since this is an entry-level tutorial, but up to you.
Fixing the line below that says click **Create component** is important though.

Reply:
fine with me, done